internal/runner/runner.go

```diff
@@ -82,10 +81,10 @@ type Runner struct {
 	// also verify the context didn't cancel. The stateCond will be broadcasted
 	// when the root context cancels.
 	stateCond       *sync.Cond
-	stateConfig     bool // config stream is connected
-	stateConfigOnce bool // true once we process config once, success or error
-	stateJobReady   bool // ready to start accepting jobs
-	stateExit       bool // true when exiting
+	stateConfig     uint64 // config stream is connected
+	stateConfigOnce uint64 // true once we process config once, success or error
+	stateJobReady   uint64 // ready to start accepting jobs
+	stateExit       uint64 // true when exiting
```

Comment:
Do these struct doc strings need to be updated to what `uint64` values mean in this case? Perhaps it's just 0 true 1 false, N "something else"?